### ISSUE TITLE
[Feature request] Exclusion

### ISSUE THREAD

**Issue by rEnr3n** [EdJoPaTo/website-stalker#14]
I'd like an option to exclude specific html element(s) within watched element(s).

**Comment by EdJoPaTo** [comment]
Currently there are `css_selector` and `regex_replacers` to limit the result.
You are seeking something like an opposite of css_selector?

Having more also raises the question of an order there in which these are run. Regex has to be last as it probably destroys the valid html syntax which the css_selectors are avoiding.

**Comment by rEnr3n** [comment]
```
  - html:
      url: https://asianload.io/videos/moms-diary-my-ugly-duckling-episode-1
      css_selector: .video-block
      regex_replacers:
        - pattern: <div class="meta">[\s\S]+</div>
          replace: ""
```

I wanted to watch all `.video-block` and remove the `.meta` inside them. I tried this regex but `</div>` matches the very last instance on the page instead of the first one. I need it to be not greedy.

**Comment by EdJoPaTo** [comment]
I not entirely happy with the implementation done in 44935b5 yet.

In order to use it the config has to be adapted. I'm not against breaking changes but I'm not sure the the config is great.
```diff
 - html:
     url: https://edjopato.de
-    css_selector: div
+    css_selectors:
+      - div
```

One possible improvement might be to allow for one entry without an array (String → Array of one String).

The syntax for removing something via CSS selector is currently a `!` in front of the CSS selector. The following config removes all the div elements with their content.
```yaml
- html:
    url: https://edjopato.de
    css_selectors:
      - "!div"
```

I'm not sure about this `!` prefix as its something different from known and common CSS selectors.

**Comment by EdJoPaTo** [comment]
I came up with a different config. Rather than assuming html from the start there can be different editors manipulating the content now. (Check e87f77e for the config change)

This way the site entry looks like this:

```yaml
- url: https://edjopato.de
  extension: html
  editors:
    - css_selector: div
    - html_prettify
```

Or when removing

```yaml
- url: https://edjopato.de
  extension: html
  editors:
    - css_selector:
        selector: div
        remove: true
    - html_prettify
```

This way the removal does not change the known css selector syntax (it doesnt introduce `!` at the start) and works well with more editors.

**Comment by EdJoPaTo** [comment]
From the example you gave this would be the way of doing it with v0.8:

```yaml
  - url: "https://asianload.io/videos/moms-diary-my-ugly-duckling-episode-1"
    extension: html
    editors:
      - css_selector: .video-block
      - css_selector:
          selector: .meta
          remove: true
      - html_prettify
```

**Comment by EdJoPaTo** [comment]
I think this issue is solved with v0.8. If there is still something that might be helpful or something not working as expected feel free to comment or create a new issue. Thanks for bringing up the idea!
